cfg_VHcc_ZLL.py

```diff
@@ -166,5 +191,30 @@
                                    bins=[0,80,150,200,300,450,700],
                                    type="variable",
                                    label="Jets $H_T$ [GeV]")]),
+        "dphi_jj_dr_jj": HistConf([ Axis(field="dijet_deltaR_low", bins=50, start=0, stop=5, label=r"$\Delta R_{jj}$, 50 < $p_{T_{\ell\ell}}$ < 150"),
+                              Axis(field="dijet_deltaPhi_low", bins=64, start=-1, stop=3.5, label=r"$\Delta \phi_{jj}$, 50 < $p_{T_{\ell\ell}}$ < 150")]),
     }
 )
+
+
+run_options = {
```

Comment:
The run options are no longer used in this config. Could you remove this part please.
Instead the `params/run_options.yaml` can be used for additional options (that are different from default in pocket-coffea.)